auslib/admin/views/forms.py

```diff
@@ -1,13 +1,14 @@
 import simplejson as json
 
 from flask_wtf import Form
-from wtforms import StringField, IntegerField, SelectField, BooleanField
+from wtforms import StringField, IntegerField, SelectField, BooleanField, TextField
```

Comment:
Well I had other idea in mind when I first tried to fix this so i guess I just forgot about it. I can remove it no problem.